File: backend-model/src/main/java/outcobra/server/model/dto/TaskDto.java
package outcobra.server.model.dto;

import outcobra.server.model.Task;
import outcobra.server.model.interfaces.MappableDto;
import outcobra.server.model.interfaces.Mapper;

public class TaskDto implements MappableDto<TaskDto, Task> {
    @Override
    public Mapper<TaskDto, Task> getMapper() {
        throw new UnsupportedOperationException("not implemented");
    }

    @Override
    public Task toEntity() {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: backend/src/main/kotlin/outcobra/server/model/dto/emptyDto/TaskDto.kt
package outcobra.server.model.dto.emptyDto

import outcobra.server.model.Task
import outcobra.server.model.mapper.MappableDto
import outcobra.server.model.mapper.Mapper

class TaskDto : MappableDto<TaskDto, Task> {
    override fun getMapper(): Mapper<TaskDto, Task> {
        throw UnsupportedOperationException("not implemented")
    }

    override fun toEntity(): Task {
        throw UnsupportedOperationException("not implemented")
    }
//TODO Implement
}